v-next/hardhat/src/internal/cli/telemetry/analytics/analytics.ts

```diff
@@ -24,25 +24,25 @@ const SESSION_ID = Math.random().toString();
 const ENGAGEMENT_TIME_MSEC = "10000";
 
 // Return a boolean for testing purposes to verify that analytics are not sent in CI environments
-export async function sendTelemetryConsentAnalytics(
-  consent: boolean,
+export async function sendTelemetryConfigAnalytics(
+  enabled: boolean,
 ): Promise<boolean> {
```

Comment:
FYI, I also renamed helper functions that dealt with telemetry permissions, like this one for example, so that they refer to whether telemetry is enabled or disabled rather than whether telemetry consent is granted or revoked. I thought this was more in line with the new task you proposed and its flags.

If you prefer to stay with the previous nomenclature outside the task, let me know. I can restore it in no time.

Reply:
makes a ton of sense